### ISSUE TITLE
Drop different variants of same module, keep only a common one

### ISSUE THREAD

**Issue by Eshaan7** [intelowlproject/IntelOwl#224]
```
['VirusTotal_v2_Get_File', 'VirusTotal_v2_Scan_File', 'VirusTotal_v3_Get_File', 'VirusTotal_v3_Get_File_And_Rescan_If_Old', 'VirusTotal_v3_Get_File_And_Scan', 'VirusTotal_v3_Get_File_And_Scan_Or_Rescan_If_Old', 'VirusTotal_v3_Scan_File']
```
These analyzers are not meant to be executed together: you should choose the most appropriate to your needs and run only one of it. 

_Originally posted by @mlodic in https://github.com/intelowlproject/IntelOwl/issue_comments/707764860_

Now that IntelOwl supports passing an analyzer configuration at runtime, both via CLI and web GUI, we could drop the various variants of the same module/service and leave only one which is most commonly used and is the recommended one. 

This would make it easier to understand for users, prevent many bugs (that are generally caused by running diff. variants together) and also easier for us to manage.

**Comment by mlodic** [comment]
Working on this + doc in the `drop_variants` branch

**Comment by mlodic** [comment]
solved PR #224
